examples/Liber/Common/Common.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="..\Proto\common.proto" Link="Protos\common.proto" />
+
+    <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPackageVersion)" />
+    <PackageReference Include="Grpc.Tools" Version="$(GrpcPackageVersion)" PrivateAssets="All" />
+  </ItemGroup>
```

Comment:
On my team, we make sure to include the .proto files in the NuGet packages as an escape hatch for advanced consumers (e.g., custom code gen options). This has been a Good Thing<sup>TM</sup> for us, and I think the ecosystem would benefit if this example did something similar.

We use something like this in the .csproj files to pack them in the .nupkg:

```xml
<ItemGroup>
  <None Include="@(ProtoBuf)">
    <CopyToOutputDirectory>Never</CopyToOutputDirectory>
    <Pack>true</Pack>
    <!-- We use MakeRelative to replicate the input directory hierarchy under the build\proto directory. -->
    <PackagePath>build\proto\$([MSBuild]::MakeRelative($(ProtoIncludeFolder), %(Identity)))</PackagePath>
  </None>
</ItemGroup>
```

In this example, `$(ProtoIncludeFile)` would be "..\Proto". (This snippet is actually from a Directory.Build.targets in our shared contracts with C# enhancements repository. If you want more details, email me at my Microsoft address and I can point you at the internal repo.)

Reply:
Interesting. I'll leave it as is for the moment. When I have more time available to research best practices for publishing NuGet packages I'll look into this more.